src/Microsoft.PowerShell.Commands.Utility/commands/utility/trace/MshHostTraceListener.cs

```diff
@@ -62,35 +62,8 @@ internal PSHostTraceListener(PSCmdlet cmdlet)
         [SecurityPermission(SecurityAction.LinkDemand)]
         protected override void Dispose(bool disposing)
         {
-#if CORECLR
             base.Dispose(disposing);
-#else
-            try
-            {
-                if (disposing)
-                {
-                    this.Close();
-                }
-            }
-            finally
-            {
-                base.Dispose(disposing);
-            }
-#endif
-        }
-
-#if !CORECLR
-        /// <summary>
-        /// Closes the dialog and then calls the base class Close
-        /// </summary>
-        [SecurityPermission(SecurityAction.LinkDemand)]
-        public override void Close()
-        {
-            // Call the base class close
-
-            base.Close();
```

Comment:
In CoreFX it is dummy method https://github.com/dotnet/corefx/blob/7da531794fe1f77668b0705502cd0658eba75889/src/System.Diagnostics.TraceSource/src/System/Diagnostics/TraceListener.cs#L173
In .Net too https://referencesource.microsoft.com/#System/compmod/system/diagnostics/TraceListener.cs,2facc6077fc08453
What is a behavior we want get?